### PR TITLE
Book operations crud

### PR DESCRIPTION
**Pull request by carolinaWolox**
## Summary
- Crud operations controller and exceptions
## Trello Card
https://trello.com/c/ZTOW2tWn/18-crear-las-operaciones-crud-de-book

### CODE REVIEW

**Review by julianvasquez-wolox — DISMISSED** [review]
LGTM! 😁

**Review by juanwongramirez — DISMISSED** [review]
Great!!

**Review by julianvasquez-wolox — DISMISSED** [review]

**Review by juanwongramirez — DISMISSED** [review]

**Review by julianvasquez-wolox — APPROVED** [review]

File: src/main/java/wolox/training/exceptions/RestExceptionController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class RestExceptionController  extends ResponseEntityExceptionHandler {
```
Comment (juanwongramirez) [inline src/main/java/wolox/training/exceptions/RestExceptionController.java]: you can to manage the class names according his function , can rename this class like RestExceptionHandler

**Review by juanwongramirez — CHANGES_REQUESTED** [review]

File: src/main/java/wolox/training/exceptions/RestExceptionController.java (hunk 1 above)
Comment (carolinaWolox) [reply src/main/java/wolox/training/exceptions/RestExceptionController.java]: It was solve last commit

**Review by juanwongramirez — APPROVED** [review]
God job

**Pull request merged**